desktop/src/app/components/navbar/warnings/modals/fix-outliers-modal.html

```diff
@@ -36,19 +48,21 @@ <h5 class="modal-title" i18n="@@navbar.taskbar.fixOutliers.header">
         </div>
     </div>
     
-    <div *ngIf="selectedValue && replaceAll">
+    <div *ngIf="isValid() && replaceAll">
         <div class="text-center alert alert-warning"
              i18n="@@navbar.taskbar.fixOutliers.all.info">
             Bitte beachten Sie: Durch diese Aktion wird der nicht in der Werteliste enthaltene Wert
-            <b>{{outlierValue}}</b> in sämtlichen Feldern aller <b>{{countAffected}}</b> Ressourcen durch den
-            ausgewählten Wert <b>{{getValueLabel(selectedValue)}}</b> ersetzt.
+            <b>{{outlierValue}}</b> in sämtlichen Feldern aller <b>{{countAffected}}</b> Ressourcen durch 
+            @if (selectedValues.length == 1) { den ausgewählten Wert } @else { die ausgewählten Werte }
```

Comment:
This might be a bit confusing for translators, but we'll have to check how it looks in Weblate. Maybe it's ok (it's definitely more elegant than working with ngIf).